src/services/ScrapboxService.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  ScrapboxNotice,
+  ScrapboxNoticeAttachment,
+} from '../@types/ScrapboxNotice';
+
+const ScrapboxService = () => {
+  const isNotifiable = (attachment: ScrapboxNoticeAttachment): boolean => {
+    const regexList = [/^日報[\s　].*/, /^下書き[\s　].*/];
```

Comment:
[nits]
```suggestion
    const regexList = [/^日報[\s　]/, /^下書き[\s　]/];
```

前方マッチさせればいいので `.*` は不要？